docs/azure-mint-mod-removal.md

```diff
@@ -0,0 +1,19 @@
+# Azure Mint Mode removal
+
+## Summary
+
+Azure is [removing support](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/update-your-applications-to-use-microsoft-authentication-library/ba-p/1257363) for the Azure Active Directory Graph API that is used to create App Registrations, Service Principals, role assignments, and credentials for supporting Mint mode in Azure. CCO will pivot existing clusters to switch away from Mint mode to Passthrough mode.
+
+## New installations
+
+For new cluster installations, Passthrough and Manual modes will be the only supported modes of operation.
+
+## Upgrades
+
+For a cluster previously installed/running in Mint mode, CCO will update existing Secrets containing the credentials of previously minted App Registrations/Service Principals with the contents of the Secret kube-system/azure-credentials (normally containing the credentials used during installation). It is required that the permissions associated with the credentials in this Secret be sufficient to be used by all in-cluster components needing to interact with Azure APIs.
+
+CCO will also try to clean up previously minted App Registrations/Service Principals while the Azure AD Graph API is still functional. If the cluster is upgraded to a version of OpenShift that no longer supports Mint mode after the Azure AD Graph API is sunset, CCO will set a condition on the associated CredentialsRequest and will not treat the error as fatal. Cleanup after the Azure AD Graph API is sunset will require manual intervention to remove the App Registrations/Service Principals that are no longer necessary.
```

Comment:
Do we want to mention the name of the condition that CCO will set?

Reply:
That would be nice esp. if it could help troubleshoot/manual clean up if needed (not sure if that's the case)

Reply:
I put the name of the condition and an example of the kind of message that would be associated with that condition.